internal/process/defunct_processes_suite_test.go

```diff
@@ -0,0 +1,26 @@
+package process_test
+
+import (
+	"testing"
+
+	. "github.com/cri-o/cri-o/test/framework"
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+)
+
+// TestDefunctProcessesForPath runs the created specs
+func TestDefunctProcessesForPath(t *testing.T) {
```

Comment:
Let's assume we're adding multiple tests to this:

```suggestion
// TestProcess runs the created specs
func TestProcess(t *testing.T) {
```